Policies/Kubernetes/host-network-ports/azurepolicy.json

```diff
@@ -0,0 +1,80 @@
+{
+  "properties": {
+    "displayName": "Control whether a Pod may use the Host Network, and allow Host Ports in a Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy controls whether pod may use the host network, and allows host ports in a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.",
+    "metadata": {
+      "version": "1.0.0-preview",
+      "category": "Kubernetes PSP",
+      "preview": true
+    },
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "enforceOPAConstraint",
+          "disabled"
+        ],
+        "defaultValue": "enforceOPAConstraint"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "allowHostNetwork": {
+        "type": "Boolean",
+        "metadata": {
+          "displayName": "Host Network Usage",
```

Comment:
Change display name to "Allow host network usage". In UI this parameter will show up like below. Currently no clue what value to provide. So in description let's say something like... "Set this value to true if pod is allowed to use host network otherwise false"

![image](https://user-images.githubusercontent.com/18429298/81898161-ff45ea00-956c-11ea-946b-6fca707bfd75.png)